gocd/templates/bash/s4s-ddog-health-check.sh

```diff
@@ -1 +1,8 @@
 #!/bin/bash
+
+/devinfra/scripts/checks/datadog/monitor_status.py --dry-run=true \
```

Comment:
what does `dry-run=true` do here?

Reply:
Oh hmm it means that on failure it doesn't actually raise an alert. Which is weird, because I copied this from the SaaS health check script. So I guess we're not actually checking the monitors there either.

Reply:
should we... check it? 🤔

Reply:
I'll look at the monitors involved and see. I think the answer is yes.